minetest/minetest
# Add a short_description to be used by mods

- I've tried to implement the changes requested here: https://github.com/minetest/minetest/pull/8966#issuecomment-533924211
- There's a new `short_description` field in item def. There's also a new `get_short_description` function for item stacks.
- This PR doesn't include the feature of seeing a different tooltip when holding shift anymore. That will, if ever, be done in a separate PR.
- This PR probably fixes parts of #8956.
- If someone thinks they can do this better than me, which is likely the cause, then please go ahead (and use the code of this PR if you wish to). Maintaining this PR / series of PRs is exhausting.

## To do

This PR is a Ready for Review.

## How to test

- Go into a minimal world.
- Hover over the mese pick and the torch pick to see their description.
- Wield the mese pick or the torch and use the `item_description` chatcommand.
